sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -44,3 +61,152 @@ def get_next_template(list_func, *args, **kwargs):
             )
         )
     return feed_element
+
+
+def deserialize_value(value, value_type):
+    if value_type in ("int", "long"):
+        value = int(value)
+    elif value_type == "boolean":
+        value = value.lower() == "true"
+    elif value_type == "double":
+        value = float(value)
+    elif value_type == "dateTime":
+        value = isodate.parse_datetime(value)
+    elif value_type == "duration":
+        value = isodate.parse_duration(value)
+    return value
```

Comment:
so here're are the types supported by service/autorest?

I'm wondering if there's any chance that unknown type occur and what would user get -- or there couldn't be such situation.

Reply:
These types are what I know from C# code (inherited from Track 1 I guess). 
Actually, when a value is not of these type, the str value will be returned. So other types are not unsupported, they're just not processed into these known data types.

Reply:
based on the information we have, this should be enough.
we could always add support later, thanks for the explanantion.